src/Dispatcher/MASA.Contrib.Dispatcher.IntegrationEvents.Dapr/README.zh-CN.md

```diff
@@ -65,3 +65,41 @@ public async Task DomeIntegrationEventHandleAsync(DomeIntegrationEvent @event)
     //todo
 }
 ```
+
+### 重试策略
+
+```C#
+builder.Services
+    .AddDaprEventBus<IntegrationEventLogService>(options=>
+    {
+        // options.MaxRetryTimes = 50;//最大重试次数, 默认：50
+        // options.RetryBatchSize = 100;//单次重试事件数量, 用于从持久化数据源获取待重试事件, 默认100
+        // options.FailedRetryInterval = 60;//持久化数据源重试停歇间隔, 默认60s
+        // options.CleaningExpireInterval = 300;//清除已过期事件停歇间隔，单位：s, 默认 300s
+        // options.ExpireDate = 24 * 3600;//过期时间，CreationTime + ExpireDate = 过期时间, 默认1天
+
+        // options.LocalFailedRetryInterval = 3;//本地队列重试停歇间隔, 默认3s
+        // options.CleaningLocalQueueExpireInterval = 60;//清除本地队列已过期事件停歇间隔，单位：s, 默认 60s
+    });
+```
+
+重试分为本地队列重试以及从持久化数据源重试两种：
+
+本地队列：
+
+特点：
+- 重试间隔短，支持秒级别重试间隔
+- 从内存获取数据，速度更快
+- 系统崩溃后，之前的本地队列不会重建，自动降级到持久化队列中重试任务
+
+持久化数据源队列：
+
+特点：
+
+- 系统崩溃后，可以从db或者其他持久化源获取重试队列，确保事件100%重试
+- 作为本地内存队列的降级方案，对db或者其他数据源压力更低
+
+在单副本情况下，两种队列的任务仅会在单个队列中执行，不会存在两个队列同时执行的情况。
+在多副本情况下，同一个任务可能会被多个副本所执行，虽然我们有做幂等，但为交付保证是 At Least Once，仍然有可能出现事件发布成功，但状态更改失败的情况，
```

Comment:
在多副本情况下，同一个任务可能会被多个副本所执行。

这个意思是所有副本都会重试？没有分配压力？

Reply:
This is controlled by the concurrent lock and State of RowVersion. If the task is processed by other instances, it will be skipped, but it is not sure whether there will be situations that are not considered to cause the task to be repeated.